test-interface/src/main/scala/scala/scalanative/testinterface/TestMain.scala

```diff
@@ -88,6 +88,15 @@ object TestMain {
 
     SignalConfig.setDefaultHandlers()
 
+    // Loading debug metadata can take up to few seconds which might mess up timeout specific tests
+    // Prefetch the debug metadata before the actual tests do start
+    if (LinktimeInfo.hasDebugMetadata) {
+      val shouldPrefetch =
+        sys.env.get("PREFETCH_DEBUG_INFO").exists(v => v.isEmpty() || v == "1")
```

Comment:
Should env be prefixed with e.g. `SN_TEST` or `SCALA_NATIVE_TEST` ?

Reply:
I could be prefixed, but don't you think it is too verbose? When it's added in CI it's fine, but when testing locally it gives a lot of opportunities for typos. On the other hand the short version breaks naming consistency. What's your opinion on that?

Reply:
If this was private to this repo, then it would be okay, but since this is a public API feature it should follow existing convention. Verbosity is annoying but hopefully we choose the right default and most user should never need to configure it.

Reply:
I know Denys always wanted it spelled out `SCALA_NATIVE_` perhaps because so many C libs use a 2 character namespacing like `tf_` for Tensorflow.

Reply:
I've added the `SCALANATIVE_TEST_` prefix to allign it with other env variables (eg. `SCALANATIVE_GC`) and added `TEST` to point out it's only applied when testing